content/en/docs/setup/upgrade/canary/index.md

```diff
@@ -59,9 +59,12 @@ istio-sidecar-injector-canary   1          3m18s
 {{< warning >}}
 Due to [a bug](https://github.com/istio/istio/issues/28880) in the creation of the `ValidatingWebhookConfiguration` during install, initial installations of Istio __must not__ specify a revision. As a temporary workaround, for Istio resource validation to continue working after removing the non-revisioned Istio installation, the `ValidatingWebhookConfiguration` must be manually pointed to the revision that should handle validation.
```

Comment:
I liked the other part of the change here though, telling the user they need to create an istiod service, I thought that was the point of the bug? It isn't enough just to change ValidatingWebhookConfiguration, you need to create a revision-less istiod service and point it the right place?

Reply:
Suggested text:

As a temporary workaround, for Istio resource validation to continue working after removing the non-revisioned Istio installation, the `istiod` service must be manually pointed to the revision that should handle validation.

Reply:
Yup, makes sense, changed. Wasn't sure if users would see the connection between the `ValidatingWebhook` and the `istiod` service but it's probably better we don't mention "modifying where the webhook points" at all.